orc8r/cloud/deploy/bare-metal-ansible/roles/magma/orc8r/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+- name: orc8r | Ensure orc8r helm repo
+  command: helm repo add orc8r {{ orc8r_helm_repo }}
+
+- name: orc8r | Update repo metadata
+  command: helm repo update
+
+- name: orc8r | Ensure namespace
+  command: kubectl create ns {{ magma_namespace }}
+  failed_when: false
+
+- name: orc8r | Ensure orc8r dirs
+  file:
+    path: "{{ item }}"
+    state: directory
+    recurse: yes
+  with_items:
+    - "{{ orc8r_config_dir }}"
+    - "{{ orc8r_charts_dir }}"
+    - "{{ orc8r_certs_dir }}"
+
+- name: orc8r | create certificates
+  include_tasks: certs.yml
+
+- name: orc8r | Template out helm chart values
+  template:
+    src: orc8r-values.yaml.j2
+    dest: "{{ charts_dir }}/{{ item }}-values.yaml"
+  with_items:
+    - orc8r
+
+- name: orc8r | template DB setup script
+  template:
+    src: db_setup.sql.j2
+    dest: "{{ orc8r_config_dir }}/db_setup.sql"
+    mode: 600
+
+- name: orc8r | copy orc8r settings secrets
+  copy:
+    src: "{{ item }}"
+    dest: "{{ orc8r_config_dir }}/{{ item }}"
+    mode: 0600
+  with_items:
+    - orc8r-configs.yaml
+    - orc8r-envdir.yaml
+
+- name: orc8r | create orc8r settings secrets
+  command: kubectl -n {{ magma_namespace }} apply -f {{ orc8r_config_dir }}/{{ item }}
+  with_items:
+    - orc8r-configs.yaml
+    - orc8r-envdir.yaml
+  
+- name: orc8r | run DB setup script
+  shell: kubectl -n mariadb exec -i mariadb-0 -- mysql -u root "--password={{ db_root_password }}" mysql < db_setup.sql
+  args:
+    chdir: "{{ orc8r_config_dir }}"
+
+- name: orc8r | precreate persistentvolumeclaims for components
+  script: make_magma_pvcs.sh {{ magma_namespace }} {{ storage_class }} ReadWriteMany {{ item.key }} {{ item.value }}
+  with_items: "{{ magma_pvcs | dict2items }}"
+
+- name: orc8r | Install/upgrade helm charts
```

Comment:
For v1.4, based on the `orc8r_deployment_type` that is set we will need to deploy the `lte-orc8r` chart (`fwa` deployment type) or `lte-orc8r` and `feg-orc8r` charts as well (`federated_fwa` deployment type). 

You can see this logic in the tf files on master in `orc8r/cloud/deploy/terraform/orc8r-helm-aws/main.tf`

Reply:
I'm glad you pointed out there are changes for 1.4 and new chart names. I will take a look at them. 1.4 is still not released, though. My goal is to provide a playbook for deploying the 1.3.x